RTX 3060 Mobile / Max-Q 2520

toccata10 opened the pull request:
extract from my nvidia.log:
Detected a NVIDIA GPU
Checking card: GeForce RTX 3060 Mobile / Max-Q with ID: 2520

toccata10 committed:
RTX 3060 Mobile / Max-Q 2520

extract from my nvidia.log:
Detected a NVIDIA GPU
Checking card: GeForce RTX 3060 Mobile / Max-Q with ID: 2520

dmanlfc commented:
it needs to go in  the manual section, where you have it is provided by the drivers themselves & gets updated.

toccata10 committed:
moved to manual

That's the name I have in my nvidia.log:
NVIDIA GeForce RTX 3060 Mobile / Max-Q
But looking at the other values, I imagine it's fine to remove the "NVIDIA" and only keep
GeForce RTX 3060 Mobile / Max-Q	2520

toccata10 commented:
ok. I moved it to the top of the file
nvidia.log:
"Detected a NVIDIA GPU
Checking card: GeForce RTX 3060 Mobile / Max-Q with ID: 2520"

Not sure if the exact spacing is important between the name and the id (neither if I need to add a "K" at the end)
